clients/ui/frontend/src/app/pages/modelCatalog/components/HardwareConfigurationTableColumns.ts

```diff
@@ -300,6 +276,20 @@ export const hardwareConfigColumns: HardwareConfigColumn[] = [
     width: 20,
     modifier: 'wrap',
   },
+  {
+    field: 'use_case',
+    label: 'Use Case',
+    sortable: (
+      a: CatalogPerformanceMetricsArtifact,
+      b: CatalogPerformanceMetricsArtifact,
+    ): number => {
+      const useCaseA = getStringValue(a.customProperties, 'use_case');
+      const useCaseB = getStringValue(b.customProperties, 'use_case');
+      return useCaseA.localeCompare(useCaseB);
+    },
+    width: 20,
+    modifier: 'wrap',
+  },
```

Comment:
Ah we did add a use case column - that's fine, I think we might want to move it elsewhere and maybe have it format the values to look nice but we can consult with UX on that, if it ships like this I'm happy